### DISCUSSION TITLE
Font and height

### DISCUSSION THREAD

**Discussion by never-failing** [ImageMagick/ImageMagick#5714] [Help]
I use the portable version for Windows + console. The problem is that I cannot figure out how to make the font size dynamic. There are photos
 
![001](https://user-images.githubusercontent.com/76694308/199773617-b42d888c-25ef-4cb2-889c-b1c6fa49c551.jpg)

На который я налаживаю текст
`convert 001.jpg -size 780x520 -channel rgba -pointsize 56 -strokewidth 2 -font AdigianaUltra.ttf -gravity center ( -background transparent -stroke #000b -fill #000a caption:"КОМПОТ ИЗ ЧЁРНОЙ СМОРОДИНЫ С АПЕЛЬСИНОМ НА ЗИМУ" -blur 3x3 -stroke none -fill #ffff caption:"КОМПОТ ИЗ ЧЁРНОЙ СМОРОДИНЫ С АПЕЛЬСИНОМ НА ЗИМУ" -flatten ) -compose dissolve -define compose:args=75 -composite -quality 100 999.png`

As a result, I get almost what I need

![999](https://user-images.githubusercontent.com/76694308/199773977-e5d5a5d1-bc02-4f66-9785-d275e840cae5.png)

The problem is that I cannot understand how to insert the `pointsize %[fx:h*95.0]` into the code and indentation from above, depending on the font size

![003](https://user-images.githubusercontent.com/76694308/199778175-d14024ea-4f4b-47f0-8751-fbbd805df264.png)

For example, if the font is 90%, then the indentation is 10%, if 80%, then the indentation is 20%, etc.

I apologize for the bad translation into English, the google translator is to blame)

**Comment by snibgo** [comment]
The units of height and width are *pixels*.

The units of pointsize are *points*, where one point is 1/72 inch.

So when calculating pointsize from height or width, you need to account for the image density (dots per inch).

Does that help?
